code/modules/events/stray_cargo.dm

```diff
@@ -7,13 +7,45 @@
 	earliest_start = 10 MINUTES
 	category = EVENT_CATEGORY_BUREAUCRATIC
 	description = "A pod containing a random supply crate lands on the station."
+	admin_setup = /datum/event_admin_setup/stray_cargo
+
+/datum/event_admin_setup/stray_cargo
+	///Admin setable override that is used instead of selecting a random location
+	var/atom/landing_turf_override
+	///Admin setable override to spawn a specific cargo pack type
+	var/pack_type_override
+
+/datum/event_admin_setup/stray_cargo/prompt_admins()
+	var/admin_targeted = tgui_alert(usr,"Aimed at current turf?", "Pod Targetting", list("Yes", "No"))
```

Comment:
If possible logically, would be good to have a "Cancel" option, and/or have it respect getting a null value in case some admin changes their mind while doing this stuff.

Also, current turf doesn't seem that descript? Might be apt to do "Aimed at turf we're on?" or something since you check for the location of usr.